code/modules/antagonists/_common/antag_datum.dm

```diff
@@ -296,9 +296,12 @@ GLOBAL_LIST_EMPTY(antagonists)
 	var/mob/chosen_one = SSpolling.poll_ghosts_for_target(check_jobban = job_rank, role = job_rank, poll_time = 5 SECONDS, checked_target = owner.current, alert_pic = owner.current, role_name_text = name)
 	if(chosen_one)
 		to_chat(owner, "Your mob has been taken over by a ghost! Appeal your job ban if you want to avoid this in the future!")
-		message_admins("[key_name_admin(chosen_one)] has taken control of ([key_name_admin(owner)]) to replace a jobbanned player.")
+		message_admins("[key_name_admin(chosen_one)] has taken control of ([key_name_admin(owner)]) to replace antagonist banned player.")
+		log_game("[key_name(chosen_one)] has taken control of ([key_name(owner)]) to replace antagonist banned player.")
 		owner.current.ghostize(FALSE)
 		owner.current.key = chosen_one.key
+	else
+		log_game("Couldn't find antagonist ban replacement for ([key_name(owner)]).")
```

Comment:
we should also warn the admins here that a jobbanned player is being allowed to do the role and warn the player they will be monitored by the admins as such